client/src/app/actions/feedActions.js

```diff
@@ -40,7 +44,7 @@ export function addPostsToFeed(data) {
   };
 }
 
-export const toggleLike = (post, id) => {
+export const toggleLike = (post, id, profileFeedFilter, feedFilter) => {
```

Comment:
Move toggle like into post actions for continuity.

Reply:
fixed!